python/style.rst

```diff
@@ -151,6 +151,66 @@ The ``exclude`` field lists paths that are not usefully linted by :command:`flak
 Auto-generated Python should not be linted (including :file:`bin/` for Stack packages with :file:`bin.src/` directories).
 We also discourage linting :file:`__init__.py` modules due to the abundance of :pep:`8` exceptions typically involved.
 
+.. _style-guide-py-ruff:
+
+Code MAY be validated with ruff
+-------------------------------
+
+The ruff_ tool may be used to validated Python source code against the portion of :pep:`8` adopted by Data Management.
+ruff_ is significantly faster than :command:`flake8` to execute and this can be useful for larger projects.
+It also is far more configurable with many built in linting extensions.
+Sometimes ruff_ will suggest fixes that can be automated.
+Use the ``--fix`` option to apply the automated fixes and then examine them before committing them to make sure they look correct.
```

Comment:
```suggestion
Use the ``--fix`` option to apply the automated fixes, and then examine them before committing them to make sure they look correct.
```

This is also redundant with the text at the end of the configuration section.  Please choose one or the other.